src/services/admin/admin.tsx

```
@@ -0,0 +1,101 @@
+    const response = await axios.get(
+      `${BASE_URL}/fields`,
+      {
+        params: { context, contextType },
+        headers: {
+          accept: 'application/json',
+          Authorization: `Bearer ${localStorage.getItem('authToken')}`,
+        },
+      }
+    );
+    // Response is an array of field objects
+    return response.data.map((field: any) => ({
```

ESLint

[error] Replace `····` with `↹↹`

(prettier/prettier)